xrpl/models/utils.py

```diff
@@ -1,9 +1,41 @@
 """Helper util functions for the models module."""
+
+import re
 from dataclasses import dataclass, is_dataclass
-from typing import Any, Dict, List, Type, TypeVar, cast
+from typing import Any, Dict, List, Optional, Pattern, Type, TypeVar, cast
+
+from typing_extensions import Final
 
 from xrpl.models.exceptions import XRPLModelException
 
+HEX_REGEX: Final[Pattern[str]] = re.compile("[a-fA-F0-9]*")
+
+# Credentials are represented in hex. Whilst they are allowed a maximum length of 64
+# bytes, every byte requires 2 hex characters for representation
+_MAX_CREDENTIAL_LENGTH: Final[int] = 64 * 2
+
+
+def validate_credential_ids(credential_list: Optional[List[str]]) -> Dict[str, str]:
+    """
+    Returns:
+        Errors pertaining to credential_ids field
+
+    Args:
+        credential_list: An optional list of input credentials
```

Comment:
These should be flipped.

Reply:
3d66a363bcb982f8ed2895e20b02e19364d295e8